doc/source/release-notes/Percona-Server-5.6.33-79.0.rst

```diff
@@ -0,0 +1,123 @@
+.. rn:: 5.6.33-79.0
+
+============================
+|Percona Server| 5.6.33-79.0
+============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.33-79.0 on
+October 18th, 2016 (Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.33-79.0/>`_
+and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.33
+<http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-33.html>`_, including
+all the bug fixes in it, |Percona Server| 5.6.33-79.0 is the current GA release
+in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source
+and free, all the details of the release can be found in the `5.6.33-79.0
+milestone at Launchpad
+<https://launchpad.net/percona-server/+milestone/5.6.33-79.0>`_.
+
+New Features
+============
+
+ |Percona Server| has implemented support for :ref:`compressed_columns` to
```

Comment:
PS has implemented support for per-column VARCHAR/BLOB compression for the XtraDB storage engine. This also features compression dictionary support, to improve compression ratio for relatively short individual rows, such as JSON data.